build/Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY . .
 RUN make go-build
 
 ####
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7-923.1669829893
```

Comment:
@dustman9000 Something I noticed.. I'm not sure dependabot will be able to determine the next version from a build string like this. We'll have to see.. it looks like they rebuilt `8.7-923` into `8.7-923.1669829893`, but there is still an original `8.7-923`.